test/integration/parameters_permissions_test.rb

```diff
@@ -1,6 +1,6 @@
 require 'integration_test_helper'
 
-class ParametersPermissionsIntegrationTest < ActionDispatch::IntegrationTest
+class ParametersPermissionsIntegrationTest < IntegrationTestWithJavascript
```

Comment:
This changes the driver that runs the test to one that supports JS (phantomjs). If these tests need JS working to pass this change is needed, but I don't think this is related here.